src/main/resources/webapp/js/controllers.js

```diff
@@ -655,6 +655,13 @@
             $scope.form = {};
             $scope.form.type = type;
             $scope.form.dto = {};
+            if ($scope.clinics.length === 1) {
+                $scope.form.dto.clinic = $scope.clinics[0];
```

Comment:
use clicnicId and add a new field to the form.dto named clinicLocation, look comments below